src/utils/releaseNotes.test.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  extractDatesFromReleasNotes,
+  monthToString,
+  monthValueToString,
+  sortReleaseNotesByDate,
+  yearValueToString,
+} from './releaseNotes';
+
+const dateObject = new Date('2023-06-29T10:50:22.8210567+00:00');
+const dates = [
+  { createdDate: '2023-06-29T10:50:22.8210567+00:00' },
+  { createdDate: '2022-06-29T10:50:22.8210567+00:00' },
+  { createdDate: '2022-05-29T10:50:22.8210567+00:00' },
+];
+
+const notes = [
+  { createdDate: '2023-06-29' },
+  { createdDate: '2020-06-29' },
+  { createdDate: '2022-06-29' },
+  {},
```

Comment:
Why the empty objects here?

Reply:
test coverage, to check what happens when createdDate is not present

Reply:
Added `createdDate` and set them as `undefined` in those empty objects to make it clearer that we are specifically looking at what we get from `createdDate`